Norm_end_to_end.Rmd

```diff
@@ -77,70 +75,106 @@ naomit <- assay(res, "raw_filled") |>
     na.omit()
 ```
 
-### PCA unsupervised 
+### PCA unsupervised
 
 Look for overall biases in batch, sample, injection index,...
 
-```{r unsupervised checks, include=FALSE}
-pca_res <- prcomp(t(log2(naomit)), scale = FALSE, center = TRUE)
+```{r unsupervised checks, include=TRUE, fig.width = 10, fig.height = 5}
+## note: scale (center and scale) on columns (features after the t() call) to
+## remove dependency on absolute abundances.
+vals <- assay(res, "raw_filled_imputed") |>
```

Comment:
I would create a centered and scaled matrix first and then pass that to the `prcomp` function (disabling any additional scaling/centering there). Simply to separate the two things a bit